apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainInAnyOrderOnlyCreatorSamples.kt

```diff
@@ -5,5 +5,16 @@ import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class IterableLikeToContainInAnyOrderOnlyCreatorSamples {
+    @Test
+    fun elementsOf(){
+        expect(listOf("A","B","C")).toContain.inAnyOrder.only.elementsOf(
+            listOf("A","B","C")
+        )
 
+        fails {
+            expect(listOf("A","B","C")).toContain.inAnyOrder.only.elementsOf(
+                listOf("A","B")
+            )
+        }
```

Comment:
```suggestion
        fails { // because not all elements found
            expect(listOf("A","B","C")).toContain.inAnyOrder.only.elementsOf(
                listOf("B", "A")
            )
        }
        
        fails { // because more elements expected than found
            expect(listOf("A","B","C")).toContain.inAnyOrder.only.elementsOf(
                listOf("B", "A", "C", "D")
            )
        }
```